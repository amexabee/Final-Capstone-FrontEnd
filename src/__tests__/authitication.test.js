import React from 'react';
import { render } from '@testing-library/react';
import Authentication from '../components/signup/authentication';

describe('Authentication', () => {
  it('renders correctly', () => {
    const { container } = render(<Authentication />);
    expect(container).toMatchSnapshot();
  });
});
