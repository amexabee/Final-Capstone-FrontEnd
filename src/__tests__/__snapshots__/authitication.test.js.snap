// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h3
      class="m-4"
    >
      Join our community.
    </h3>
    <div
      class="d-flex flex-column"
    >
      <form>
        <div
          class="form-outline mb-4"
        >
          <input
            class="form-control"
            id="email"
            placeholder="Email"
            type="email"
            value=""
          />
        </div>
        <div
          class="form-outline mb-4"
        >
          <input
            class="form-control"
            id="password"
            placeholder="Password"
            type="password"
            value=""
          />
        </div>
        <div
          class="form-outline mb-4"
        >
          <input
            class="form-control"
            id="password-confirmation"
            placeholder="Confirm password"
            type="password"
            value=""
          />
        </div>
        <div
          class="d-flex justify-content-center"
        >
          <button
            class="btn btn-success mb-4"
            type="submit"
          >
            Sign Up
          </button>
        </div>
      </form>
      <p
        class="mt-4 align-self-center"
      >
        Already Have an Account?
      </p>
      <button
        class="btn btn-success align-self-center"
        type="button"
      >
        Sign In
      </button>
    </div>
  </div>
</div>
`;
